src/cascadia/TerminalControl/TSFInputControl.cpp

```diff
@@ -0,0 +1,380 @@
+#include "pch.h"

```

Comment:
This one needs a license preamble.

Reply:
Good catch, it's missing in the .h too, will add it.